pkg/controllers/osp/osp_controller.go

```diff
@@ -76,13 +78,16 @@ func Add(mgr manager.Manager, log *zap.SugaredLogger, namespace string, workerCo
 		log:             log,
 		defaultOSPFiles: defaultOSPFiles,
 		namespace:       namespace,
+		ospNamespace:    ospNamespace,
 	}
 
 	c, err := controller.New(ControllerName, mgr, controller.Options{Reconciler: reconciler, MaxConcurrentReconciles: workerCount})
 	if err != nil {
 		return err
 	}
 
+	log.Info("Reconciling OSP resource..")
```

Comment:
Actually this function doesn't reconcile any OSP resources it only creates the controller.

Reply:
Sorry just added this for testing something, removed.